101-sql-vm-new-storage-ultrassd/README.md

```diff
@@ -0,0 +1,33 @@
+# Create a SQL Virtual Machines with Performance-Optimized Storage Settings using Ultra SSD For SQL Log Files
+
+
+Before deploying the template you must have the following
+
+1. **Virtual Network** and **Subnet** a Virtual Network within the same resource group and a subnet must exist in the same region as the VM
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-sql-vm-new-storage-ultrassd%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-sql-vm-new-storage-ultrassd%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+`Tags: Azure, SQL, VirtualMachine, Performance, StorageConfiguration, UltraSSD`
+
+## Solution overview and deployed resources
+
+This deployment will create a VM running SQL Server with SQL Data, Log and Temp DB file separate into different drive. 
```

Comment:
fix grammar

This deployment will create a VM running SQL Server with SQL Data, Log and Temp DB file on different drives.